### ISSUE TITLE
access first positional argument of initial folder to show

### ISSUE THREAD

**Issue by metasong** [mgunyho/tere#61]

**Comment by mgunyho** [comment]
Hi, do you mean that `tere /some/folder` would start `tere` in the path `/some/folder`?

This would be pretty easy to do, but I think the goal of `tere` is that it should be faster/more convenient to open `tere` and then navigate to that folder path inside the app, than to type it out. It should require the same number or less keystrokes, and you get instant visual feedback on the possible options.

Do you have a use case in mind where having the starting path as an argument would be better? Perhaps if `/some/folder` is the output of some other command, but that feels like quite a contrived example.

**Comment by metasong** [comment]
yes, you can have a folder path stored in a env variable, i.e. in pwsh, $env:applicationdata ...., and you know the file is in one of subfolder, you just want to start from there

**Comment by metasong** [comment]
also we have already logged the pathes user navigated, we can make full uses of it, to just type part of the folder name and interpret it as the fuzzy search folder name  parameter to jumping.
